src/main/java/io/javaalmanac/snippets/time/DaylightSavingTime.java

```diff
@@ -7,12 +7,12 @@
 import java.time.zone.ZoneRules;
 
 /**
- * The Java time API uses the [tz
- * database](https://en.wikipedia.org/wiki/Tz_database) to handle DST rules for
+ * The Java time API uses the
+ * <a href="https://en.wikipedia.org/wiki/Tz_database">tz database</a> to handle DST rules for
```

Comment:
Unfortunately the rendering pipeline need markdown here as this text is automatically embedded in md pages in the java-almanac repo. See `io.javaalmanac.snippets.site.Generator` in this repo.